content/en/container-orchestration.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Container Orchestration
+status: Draft
+category: Concept
+---
+
+## What it is
+[Container](/container/) orchestration refers to managing and automating the lifecycle of containerized applications in dynamic environments. 
+It's executed through a [cloud native technology](/cloud-native-tech/) (in most cases, [Kubernetes](/kubernetes)) that enables deployments, (auto)scaling, auto-healing, and monitoring. 
+Orchestration is a metaphor. 
+The orchestration tool conducts containers like a music conductor, ensuring every container (or musician) does what it should. 
+
+## Problem it addresses 
+The rise of [cloud native apps](/cloud-native-apps/) and technologies and the fast-changing demands of modern businesses introduced a new set of problems. 
```

Comment:
```suggestion

```
This sentence has been bothering me. It doesn't really flow with the rest. Thinking about it today, I don't think we need it. It's a little fluffy and the other two are to the point. Ok if we delete it?